app/controllers/admin/deploy_groups_controller.rb

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
 class Admin::DeployGroupsController < ApplicationController
   before_action :authorize_admin!
-  before_action :authorize_super_admin!, only: [:create, :new, :update, :destroy, :deploy_all, :deploy_all_now, :edit]
-  before_action :deploy_group, only: [:show, :edit, :update, :destroy, :deploy_all, :deploy_all_now]
+  before_action :authorize_super_admin!, only: [:create, :new, :update, :destroy, :deploy_all, :deploy_all_now,
+                                                :create_all_stages, :edit]
+  before_action :deploy_group, only: [:show, :edit, :update, :destroy, :deploy_all, :deploy_all_now, :create_all_stages]
```

Comment:
none of these stages were changed ... why do they all need the before_filter now ?